apps/communications/dialer/test/unit/keypad_test.js

```diff
@@ -801,6 +801,8 @@ suite('dialer/keypad', function() {
         test('0# is ignored', function(done) {
           subject._getSpeedDialNumber('0').then(function(number) {
             assert.ok(false, 'the promise should be rejected');
+          }, function(error) {
+            assert.equal(error, 'noContactsWereFound');
```

Comment:
Could we add a spy to this test maybe to check that the ConfirmDialog gets called with noContactsFoundDialogOk?